hometask_10/1/main.js

```diff
@@ -1,4 +1,4 @@
-document.addEventListener("DOMContentLoaded", function() { 
+window.addEventListener('load', (event) => {
```

Comment:
why do you wait the whole window instead of waiting one image?
`image.addEventListener('load'...)`